### PR TITLE
fix(trailhead): added ToS link to emails

### LINKED ISSUE #1327: Update our email footers with most recent legal recommendations
Legal would like us to align our footers with marketing's. Here is a document with the footer recommendations they've made for each type of email being sent:

https://docs.google.com/document/d/1Ozh7ZZC0jL8doz3BKT583_8oi7iVmHeAdZ9qLIg_Vg8/edit?ts=5cedc017

### PR DESCRIPTION
**Pull request by dannycoates**
closes #1327

### CODE REVIEW

File: packages/fxa-auth-server/lib/senders/partials/base/base.html [hunk 1]
```diff
@@ -27,7 +27,9 @@
   <td valign="top">
     <p style="font-family: sans-serif; font-weight: normal; margin: 0; text-align: center; color: #737373; font-size: 11px; line-height: 18px; width: 310px !important; word-wrap:break-word">Mozilla. 331 E Evelyn Ave, Mountain View, CA 94041
     <br />
-    <a href="{{{privacyUrl}}}" style="color: #0a84ff; text-decoration: none; font-family: sans-serif; font-size: 11px; line-height: 18px;">{{t "Mozilla Privacy Policy" }}</a></p>
+    <a href="{{{privacyUrl}}}" style="color: #0a84ff; text-decoration: none; font-family: sans-serif; font-size: 11px; line-height: 18px;">{{t "Mozilla Privacy Policy" }}</a>
+    <br />
+    <a href="https://www.mozilla.org/en-US/about/legal/terms/services/" style="color: #0a84ff; text-decoration: none; font-family: sans-serif; font-size: 11px; line-height: 18px;">{{t "Firefox Cloud Terms of Service"}}</a></p>
```
Comment (clouserw) [inline packages/fxa-auth-server/lib/senders/partials/base/base.html]: I think the links should omit `en-US` so it can be auto-detected

Comment (dannycoates) [reply packages/fxa-auth-server/lib/senders/partials/base/base.html]: whoops

File: packages/fxa-auth-server/lib/senders/partials/base/base.html [hunk 2]
```diff
@@ -27,7 +27,9 @@
   <td valign="top">
     <p style="font-family: sans-serif; font-weight: normal; margin: 0; text-align: center; color: #737373; font-size: 11px; line-height: 18px; width: 310px !important; word-wrap:break-word">Mozilla. 331 E Evelyn Ave, Mountain View, CA 94041
     <br />
-    <a href="{{{privacyUrl}}}" style="color: #0a84ff; text-decoration: none; font-family: sans-serif; font-size: 11px; line-height: 18px;">{{t "Mozilla Privacy Policy" }}</a></p>
+    <a href="{{{privacyUrl}}}" style="color: #0a84ff; text-decoration: none; font-family: sans-serif; font-size: 11px; line-height: 18px;">{{t "Mozilla Privacy Policy" }}</a>
+    <br />
+    <a href="https://www.mozilla.org/about/legal/terms/services/" style="color: #0a84ff; text-decoration: none; font-family: sans-serif; font-size: 11px; line-height: 18px;">{{t "Firefox Cloud Terms of Service"}}</a></p>
```
Comment (shane-tomlinson) [inline packages/fxa-auth-server/lib/senders/partials/base/base.html]: I think updates here are supposed to cause all of the actual templates to be re-created.

**Comment by clouserw** [comment]
@dannycoates shane mentioned the txt files aren't updated either and I can confirm looking in the directory there are a bunch of them

File: packages/fxa-auth-server/scripts/template-version-bump.js [hunk 3]
```diff
@@ -27,12 +27,12 @@ const versions = require(`../${TEMPLATE_DIR}/${VERSIONS_FILE}`);
 
 const stagedTemplates = cp.execSync('git status --porcelain', { cwd: ROOT_DIR, encoding: 'utf8' })
   .split('\n')
-  .filter(line => line.match(`^[AM]. ${TEMPLATE_DIR}/\\w+`))
+  .filter(line => line.match(`^[AM]. packages/fxa-auth-server/${TEMPLATE_DIR}/\\w+`))
```
Comment (shane-tomlinson) [inline packages/fxa-auth-server/scripts/template-version-bump.js]: Thanks for this update too!

**Pull request merged**